src/main/java/com/example/tasker/Task.java

```diff
@@ -0,0 +1,88 @@
+package com.example.tasker;
+
+//import org.springframework.data.annotation.Id;
+import javax.persistence.*;
+
+@Entity
+public class Task {
+
+   @Id
+   @GeneratedValue(strategy = GenerationType.IDENTITY)
+   private Long id;
+   private String name;
+   private String description;
+   private Boolean realized;
+   private Boolean archived;
+
+   @Enumerated(EnumType.STRING)
+   private Category category;
+
+   public Task() {
+      this.id = id;
```

Comment:
W tym konstruktorze przypisujesz wartości z obiektu same do siebie :P Nie za bardzo ma to sens. Defaultowy konstruktor tutaj powinien być pusty